### PR TITLE
Things to do card

### PR DESCRIPTION
**Pull request by cmonez**
Made a basic card component and carousel. Tested some things for the card component. @krisdo @andyoe0102 @samasastudio

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const app = require('./app');
 
-app.listen(3000, () => {
```
Comment (krisdo) [inline server/server.js:3]: Why are there two different ports?

File: client/components/sampleDataForTests.js [hunk 2]
```diff
@@ -235,9 +235,237 @@ const hostAndRoomsNotEntirePlace = {
...
+  {
+    activityPhoto: 'https://picsum.photos/150/100',
+    activityDescription: 'Est autem aliquam minus.',
+    pricePerPerson: 166,
+    stars: 1,
+  },
+];
+
 export {
   sampleDescriptions,
```
Comment (krisdo) [inline client/components/sampleDataForTests.js:466]: Like the es6!

**Comment by andyoe0102** [comment]
Looks good!

File: server/server.js (hunk 1 above)
Comment (cmonez) [reply server/server.js:3]: Had to change one. This is the old port i was using before we agreed

**Pull request merged**